react-native-scripts/template/README.md

```diff
@@ -142,6 +143,34 @@ You can optionally use a [plugin for your IDE or editor](https://flow.org/en/doc
 
 To learn more about Flow, check out [its documentation](https://flow.org/).
 
+## Adding ESLint and Prettier
+
+[ESLint](http://eslint.org/) is a configurable JavaScript linter. ESLint can help standardize a style guide across the team.
+
+To add eslint to a Create React Native App project, follow these steps:
+
+1. Run `npm install --save-dev eslint eslint-config-react-native`
+2. Add `"lint": "eslint ."` to the `scripts` section of your `package.json`.
+3. Add the following to the `.eslintrc` file in the project root which is used to configure how eslint should run,
+
+```json
+{
+  "extends": ["react-native"]
+}
+```
+
+Please see the repo of [eslint-config-react-native](https://github.com/kunalgolani/eslint-config/tree/master/react-native) to know what rules are enabled with this configuration.
```

Comment:
Refer to the [eslint-config-react-native repoistory]([eslint-config-react-native](https://github.com/kunalgolani/eslint-config/tree/master/react-native) to know what rules are enabled with this configuration.) to see the exact rules that are enabled with this configuration.